contracts/abstracted_account.algo.ts

```diff
@@ -146,6 +147,21 @@ export class AbstractedAccount extends Contract {
     );
   }
 
+  /**
+   * check whether the plugin can be used
+   * 
+   * @param plugin the plugin to be rekeyed to
+   * @returns whether the plugin can be called via txn sender or globally
+   */
+  @abi.readonly
+  arc58_canCall(plugin: AppID): boolean {
```

Comment:
I'm wondering if we should also pass in the caller as a parameter, rather than always using the sender of `arc58_canCall`. For example, an app might want to check "can this other app call this plugin"?